bamboo/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ## Dockerfile to create integration/unit test environment
-FROM node:16.19.0-buster
-RUN apt update && npm config set unsafe-perm true &&\
```

Comment:
cool to see unsafe-perm go away here and elsewhere... what was it doing before that's changed?

Reply:
NPM changed how it's invoking things in v7, instead of using the uid specified by the user config (nobody in the case of root) it uses the permissions from the folder, which works just fine for our use case. 

Also, notably, they *dropped* support for this option when those changes were made.